### PR TITLE
Wip master

### PR DESCRIPTION
**Pull request by AnnJohn**

### CODE REVIEW

**Comment by jordanhudgens** [comment]
@AnnJohn Added metadata

File: README.md [hunk 1]
```diff
@@ -8,18 +8,23 @@
...
 6. Use an instance variable from a controller action in a template
-7. Iterate ove ran ActiveRecord collection in ERB
+7. Iterate over an ActiveRecord collection in ERB
 8. Build a classical index action/view
 
-## Notes
+## RESTful Routes
 
-This is a lab to build /students
+In this lab you will walk through how to build each element required to render a list of students out in the browser. The RSpec/Capybara tests for this lab can be found in ```spec/features```, to run the tests, run the following command in the terminal ```bundle exec rspec spec/features/```, both tests should be passing to complete the lab.
```
Comment (jmburges) [inline README.md]: No need to mention how to run the tests. They've been doing it for a while :)

File: spec/features/student_index_spec.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+feature 'Route to view' do
```
Comment (jmburges) [inline spec/features/student_index_spec.rb]: switch to describe/it

File: spec/features/student_index_spec.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  scenario 'has an index page' do
+    visit students_path
```
Comment (jmburges) [inline spec/features/student_index_spec.rb]: haven't covered path yet so "/students"

File: spec/features/student_index_spec.rb [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    expect(page.status_code).to eq(200)
+  end
+end
+
+feature 'Multiple students are shown' do
+  scenario 'on the index page' do
+    Student.create!(first_name: "Daenerys", last_name: "Targaryen")
+    Student.create!(first_name: "Lindsey", last_name: "Stirling")
+
+    visit students_path
```
Comment (jmburges) [inline spec/features/student_index_spec.rb]: "/students"

**Comment by jmburges** [comment]
all yours

**Comment by jordanhudgens** [comment]
@AnnJohn This one has the new changes, I also updated the `wip-solution` branch as well.

**Pull request merged**